src/components/DpConfirmDialog/DpConfirmDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <dp-modal
+    ref="confirmDialog"
+    content-classes="w-2/3 xl:w-1/3"
+    :data-cy="`confirmDialog:${dataCy}`"
+    aria-describedby="dialogDescription"
+    @modal:toggled="onModalToggled">
+    <p
+      id="dialogDescription"
+      class="font-normal break-words mb-3">
+      {{ message }}
+    </p>
+    <dp-button-row
+      primary
+      :primary-text="confirmButtonText"
+      secondary
+      @primary-action="handleConfirm(true)"
+      @secondary-action="handleConfirm(false)" />
+  </dp-modal>
+</template>
+
+<script lang="ts">
+import { de } from '~/components/shared/translations'
+import DpButtonRow from '~/components/DpButtonRow'
+import DpModal from '~/components/DpModal'
+
+export default {
+  name: 'DpConfirmDialog',
+
+  components: {
+    DpButtonRow,
+    DpModal
+  },
+
+  props: {
+    dataCy: {
```

Comment:
nonblocking, just saying: someday if demosplan-ui uses composition api, we could make this a composable, as props.dataCy is literally used everywhere.